internal/build/phase_config_provider.go

```diff
@@ -244,6 +245,12 @@ func WithRoot() PhaseConfigProviderOperation {
 	}
 }
 
+func WithoutPrivilege(uid int, gid int) PhaseConfigProviderOperation {
```

Comment:
```suggestion
func WithUser(uid int, gid int) PhaseConfigProviderOperation {
```
More descriptive and accurate, I think